lua/methods/coalitions.lua

```diff
@@ -1,11 +1,122 @@
 --
--- RPC unit actions
+-- RPC coalition actions
 -- https://wiki.hoggitworld.com/view/DCS_singleton_coalition
 --
 
 local GRPC = GRPC
 local coalition = coalition
 
+local skill = {
+  [0] = "Random",
+  [1] = "Average",
+  [2] = "Good",
+  [3] = "High",
+  [4] = "Excellent",
+  [5] = "Player",
+  Random = 0,
+  Average = 1,
+  Good = 2,
+  High = 3,
+  Excellent = 4,
+  Player = 5
+}
+
+--local altitudeType = {
+--  [1] = "BARO",
+--  [2] = "RADIO",
+--  BARO = 1,
+--  RADIO = 2
+--}
+
+local createGroundUnitsTemplate = function(unitListTemplate)
```

Comment:
Nit :D (feel free to ignore)

```suggestion
local function createGroundUnitsTemplate(unitListTemplate)
```

Reply:
Plenty of PRs to come so will do this as part of them.